### ISSUE TITLE
Need more early uses for glass

### ISSUE THREAD

**Issue by naed21** [StrangeLoopGames/EcoSuggestions#1633]
Despite Glassworking unlocking around the same time as pottery, the only thing you can make until level 4 is just plain glass (no electronics or steel available for several days). This can make leveling up glassworking quite boring, even worse you can't use it to increase your housing score like you can with pottery (plant pots).

It'd be great if glassworking got something at level 2 or level 3 that can be used for housing score and didn't require late game materials. Maybe a glass vase that has flours in it when placed. A glass chandelier could be another option.

**Comment by naed21** [comment]
Been playing some more, I'm still pretty disappointed with glassworking. I feel I'd been much more useful if I took potery or carpentry.

There's only 3 options for building with glass and since there are already window options for most stuff I feel like it'd be better if glass wasn't a block and was just an item you can stick in a chest. Maybe if that change was made you can double the amount of glass is needed by everything and double the amount of glass produced per project but half the job speed so you gain exp faster.

It just needs something to fill that gap between when you first get the skill book and when you're actually needed at electronics.

**Comment by jemmotion** [comment]
I totally agree I feel like I am making a ton of glass just to level up and not using it for anything really because aesthetically it doesn't look good with any of the early buildables. Plus would be nice to have glass options like different kinds of glass blocks, ex. glass frosted block for shower, bubble glass block used for basement window, glass table top w either wood or metal frame, mirrors for bathroom or tall mirror for bedroom, glass animal/insect statues.
